banyan-utils/src/tags.rs

```diff
@@ -18,6 +18,7 @@ use std::{
 use vec_collections::VecSet;
 
 #[derive(Debug, Clone)]
+#[allow(clippy::upper_case_acronyms)]
```

Comment:
clippy 1.51

Reply:
Can we do this somehow at top level? I find clippy a bit intrusive recently.